src/modules/services_manager_service.rb

```diff
@@ -180,6 +180,7 @@ def extract_services
         service_names.zip(ss).each do |name, s|
           sh = services[name] # service hash
           sh[:enabled] = s && s.enabled?
+          sh[:enabled] = s && (s.enabled? || !!(s.socket && s.socket.enabled?))
```

Comment:
line above should be removed.

Reply:
second and bigger problem is https://trello.com/c/fZC7vbpA/108-3-fate319428-mu-display-correct-status-information-on-services-list#comment-5b30e4a75456b7279ca89c4a as we need to differentiate between enabled and Start on Demand and show to user different string. ( that is reason why card was on blocked, as it does not match description of card, but in the end conclusion is to use this special string for socket )

Reply:
We agreed to just fix the "Enable/Disable" column as a MU and improve the text for SP1.

Reply:
but see kobliha answer there. They want new text in GA and even PMs wants it.